docs/core/install/localized-intellisense.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Install localized IntelliSense files
+description: Learn how to set up your development machine to use localized IntelliSense files.
+author: mairaw
+ms.author: mairaw
+ms.date: 12/12/2019
+---
+# How to install localized IntelliSense files for .NET Core projects
```

Comment:
Should we make it clear this is only for Windows?

Reply:
This is even only for VS, right? Or would this work for VS Code too?

Reply:
Cc @carlossanlop 

Not sure on VS Code part.

Reply:
I'm not sure if this is specific for Visual Studio, because the zip file with the localized xmls needs to be extracted in this folder:

`C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\3.0.0\ref\netcoreapp3.0\`

But I only know how to change the IntelliSense language in Visual Studio. I don't know about VS Code or other IDEs.

Reply:
My suggestion is to mention these instructions are specific for Visual Studio (which is Windows by default). I believe Maira already has that information mentioned in the document.

Thoughts, @vivmishra , @mairaw?

Reply:
I think so. We can investigate if this would work for VS Code too and edit the article if needed. 
This might work for VS for Mac too(?), but I currently don't have my Mac configured to test this. 
I'll just change the title to say Visual Studio as well and that should make it pretty clear for now. Introduction and prereqs already list VS.